src/libs/ajax.js

```diff
@@ -655,6 +658,39 @@ const Buckets = signal => ({
 })
 
 
+const GoogleBilling = signal => ({
+  listAccounts: async () => {
+    if (!hasBillingScope()) {
+      reportError('Terra does not have access to query your Google Cloud Billing information')
+    }
+    const response = await fetchGoogleBilling('billingAccounts', _.merge(authOpts(), { signal }))
+    const result = await response.json()
+    return result.billingAccounts
+  },
+
+  listProjectsWithAccounts: async billingAccountNames => {
+    return _.fromPairs(
```

Comment:
This works, but it feels like an obvious place where `_.flow()` would work well. I'm going to experiment with that.